README.md

```diff
@@ -350,6 +350,36 @@ Future.reject('it broke')
 //> Left('it broke')
 ```
 
+#### finally
+##### `#finally :: Future a b ~> Future a c -> Future a b`
+##### `.finally :: Future a c -> Future a b -> Future a b`
+
+Run a second Future after the first ends, despite of it rejecting or resolving.
```

Comment:
This doesn't quite make sense. I suggest:

_Run a second Future after the first ends (successfully or unsuccessfully)._